app/src/main/java/com/aptoide/uploader/NotificationApplicationView.java

```diff
@@ -1,38 +1,58 @@
 package com.aptoide.uploader;
 
-import android.app.Application;
-import android.app.Notification;
 import android.app.NotificationChannel;
 import android.app.NotificationManager;
 import android.app.PendingIntent;
+import android.app.Service;
 import android.content.Context;
 import android.content.Intent;
 import android.os.Build;
+import android.os.IBinder;
 import android.util.Log;
+import androidx.annotation.Nullable;
 import androidx.core.app.NotificationCompat;
+import androidx.core.app.ServiceCompat;
 import com.aptoide.uploader.apps.UploadManager;
 import com.aptoide.uploader.apps.view.NotificationPresenter;
 import com.aptoide.uploader.apps.view.NotificationView;
 import io.reactivex.Observable;
 import io.reactivex.subjects.BehaviorSubject;
 
-public abstract class NotificationApplicationView extends Application implements NotificationView {
+public class NotificationApplicationView extends Service implements NotificationView {
```

Comment:
This should probably be renamed since it is now a service (e.g. NotificationService?)